widget/embedded/src/components/CustomCollapsible/CustomCollapsible.tsx

```diff
@@ -12,6 +12,7 @@ import {
 export function CustomCollapsible(props: PropsWithChildren<PropTypes>) {
   const {
     open,
+    outlined = true,
```

Comment:
Do we really need this property? Currently, when the custom destination is open, there's no border around it. Please review the next branch. I believe the UI design is incorrect.

Reply:
Removed